cmd/run/run_test.go

```diff
@@ -1291,6 +1296,7 @@ func TestRunCommandConfigIsMerged(t *testing.T) {
 		require.Equal(t, "1ms", viper.GetString("dispatch-throttling-frequency"))
 		require.Equal(t, "120", viper.GetString("dispatch-throttling-threshold"))
 		require.Equal(t, "130", viper.GetString("dispatch-throttling-max-threshold"))
+		require.Equal(t, "120", viper.GetString("max-condition-evaluation-cost"))
```

Comment:
Here we are doing `viper.GetString(flag)`, but below we are doing `viper.GetUint64(configField)`. Why the difference? Shouldn't they remain consistent? Are we sure the `viper.GetUint64(configField)` works as well? It would be good to add a test for that as well.

Reply:
This was a case of "doing the same thing being done the line above it". Nice catch. Added both.